### PR TITLE
(fix) Improvements to workspace system styling (supports O3-3246)

### PR DESCRIPTION
**Pull request by brandones**
REQUIRES https://github.com/openmrs/openmrs-esm-core/pull/1046

## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Updates use of the workspace system to reflect the changes in https://github.com/openmrs/openmrs-esm-core/pull/1046

## Screenshots

See https://github.com/openmrs/openmrs-esm-core/pull/1046

### CODE REVIEW

**Review by denniskigen — APPROVED** [review]
I welcome the simplification! Thanks, @brandones. I'm curious to see how this works on an actual tablet device. @donaldkibet is it possible to get Josh to test this out?

File: packages/esm-patient-chart-app/src/root.scss [hunk 1]
```diff
@@ -81,6 +81,6 @@ html[dir='rtl'] {
         border-right: 0.25rem solid var(--brand-01);
         border-left: unset;
       }
-  }
+  }
```
[the remark this answers is not in the document]
Comment (brandones) [reply packages/esm-patient-chart-app/src/root.scss]: Bummer we still don't have scss auto-formatting I guess?

**Comment by brandones** [comment]
Build is failing due to form-lib needing to be updated with the latest OpenmrsDatePicker update.

**Comment by denniskigen** [comment]
Should I expect to see the side rail on desktop?

CleanShot 2024-06-26 at 11  44 12@2x

**Comment by denniskigen** [comment]
The bottom nav should have a top border on tablet: https://zeroheight.com/23a080e38/p/948cf1-siderail-and-bottom-nav

CleanShot 2024-06-26 at 11  47 06@2x

**Comment by brandones** [comment]
> The bottom nav should have a top border on tablet https://zeroheight.com/23a080e38/p/948cf1-siderail-and-bottom-nav:

Whoops! I'll need to fix that in core. Separate PR.

> Should I expect to see the side rail on desktop?

Did you pull the latest? I did a screw-up
image

**Review by denniskigen — APPROVED** [review]
LGTM apart from the top border on the bottom (which I imagine is a change to make in Core). Thanks, @brandones! Also, I think the viewport height means it should work very well on tablet (based on my responsive mode testing).

**Comment by brandones** [comment]
https://github.com/openmrs/openmrs-esm-core/pull/1055

**Comment by denniskigen** [comment]
Looks great on iPad mini @brandones @gracepotma

https://github.com/openmrs/openmrs-esm-patient-chart/assets/8509731/94700666-1b47-4a8a-916e-4496343855e1

**Comment by denniskigen** [comment]
I've tested on my 8" android tablet and it looks like this fixes other long-standing responsiveness issues such as the action buttons in the drug order form not being clickable. @donaldkibet, could you and your team verify this?

**Comment by denniskigen** [comment]
Related https://openmrs.atlassian.net/browse/O3-2946

**Pull request merged**